src/parser/element.js

```diff
@@ -91,16 +90,12 @@ class Element {
       debug('Element %s is not allowed within %j', qname, this.nsName);
     }
     
-    if (ElementType) {
-      child = new ElementType(nsName, attrs, options);
-      child.nsURI = qname.nsURI;
-      child.targetNamespace = attrs.targetNamespace || this.getTargetNamespace();
-      debug('Element created: ', child);
-      child.parent = parent;
-      stack.push(child);
-    } else {
-      this.unexpected(nsName);
```

Comment:
removing this error block seems to be unrelated to this change

Reply:
That's true...

Unfortunately, because my change reduces the test coverage very slightly, which causes Travis CI to fail.
So I was forced to remove some lines somewhere in the code to get it into positive.

If you have a better solution for this issue, feel free to suggest a change. Since I am not very deep into the code I cannot do that.